packages/cli/src/platform/login.ts

```diff
@@ -85,10 +92,15 @@ export class Login implements Command {
 
 const generateAuthSigninUrl = async (params: { connection: string; redirectTo: string }) => {
   const prismaClientVersion = await getInstalledPrismaClientVersion().catch(() => null)
-  const state = { client: `prisma@${prismaClientVersion}`, ...params }
+  const state = {
+    client: `prisma@${prismaClientVersion}`,
+    signature: await checkpoint.getSignature(),
```

Comment:
@Jolg42  what happens if this fails?

Reply:
I added a catch because in theory it can fail, it will be rare